src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java

```diff
@@ -52,8 +52,7 @@ public void destroy() {
 
 	@Given("a user clicks on Capture Vitals link from Patient dashboard")
 	public void loadVitalsPage() {
-		patientCaptureVitalsPage = (PatientCaptureVitalsPage) dashboardPage
-				.goToPatientCaptureVitalsPage().waitForPage();
+		patientCaptureVitalsPage = (PatientCaptureVitalsPage) dashboardPage.goToPatientCaptureVitalsPage().waitForPage();
```

Comment:
@sherrif10 , I have attended to the review you left for me.

Reply:
@sherrif10 and @kdaud  are there any more genuine reasons that could be stopping this pull request from being merged into master?